addons/website_sale/static/src/js/website_sale.js

```diff
@@ -6,7 +6,6 @@ import "@website/libs/zoomodoo/zoomodoo";
 import { ProductImageViewer } from "@website_sale/js/components/website_sale_image_viewer";
 import VariantMixin from "@website_sale/js/sale_variant_mixin";
 
-
```

Comment:
useless diff

Reply:
It's not like it's modifying a line potentially impacting the history for no good reason, it's rightfully removing one line :shrug: I would not bother to remove that kind of "useless diff" :shrug: (but whatever)

Reply:
Indeed, you can discard this comment :)